packages/wrangler/src/pages/dev.ts

```diff
@@ -130,7 +130,7 @@ export function Options(yargs: CommonYargsArgv) {
 			},
```

Comment:
Mind also updating D1 with an example while you're here?

Reply:
done 🙂